easybuild/easyconfigs/p/PGI/PGI-17.10-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '17.10'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
```

Comment:
@boegel Worth adding a note here about the need to download manually? I know it's implicit owing to the lack of a `sources_url` but...